Triangulation_3/include/CGAL/IO/Triangulation_file_input.h

```diff
@@ -88,10 +87,9 @@ std::istream& file_input(std::istream& is, Tr2 &tr,
   for (std::size_t j=0 ; j < m; j++) {
     Cell1 c;
     if(!(is >> c)) return is;
-    if(!update_cell(c, *(C[j]))) {
-      is.setstate(std::ios_base::failbit);
-      return is;
-    }
+    Cell_handle ch=tr.tds().create_cell(convert_cell(c));
```

Comment:
Have you tested? I am pretty sure that is wrong: cells are already created in line 85:
```
 tr.tds().read_cells(is, V, m, C);
```

Reply:
It works in the demo :(

Reply:
Not really a proof.   You might create cells that are not needed and not interconnected with other cells.

Reply:
Yes, I think there were simply too many cels in the created c3t3 and I did not see.